### PR TITLE
DDCNL-2062a Fix for link on main address tile

### PR DESCRIPTION
**Pull request by AndrewArkless**

### CODE REVIEW

File: test/views/html/address/MainAddressViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import models.{Country}
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.test.FakeRequest
+import uk.gov.hmrc.renderer.TemplateRenderer
+import util.{BaseSpec, Fixtures}
+import util.UserRequestFixture.buildUserRequest
+import views.html.ViewSpec
+import views.html.cards.personaldetails.MainAddressView
+
+class MainAddressViewSpec extends ViewSpec with BaseSpec with MockitoSugar {
```
Comment (jpatton638) [inline test/views/html/address/MainAddressViewSpec.scala:31]: Not sure if ViewSpec already extends BaseSpec, could you check?

**Review by oc-ste — APPROVED** [review]

File: test/views/html/address/PostalAddressViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import util.UserRequestFixture.buildUserRequest
+import views.html.ViewSpec
+import views.html.cards.personaldetails.{PostalAddressView}
+
+class PostalAddressViewSpec extends ViewSpec with BaseSpec with MockitoSugar {
+  override implicit lazy val app = localGuiceApplicationBuilder().build()
+
+  lazy val view = injected[PostalAddressView]
+
+  implicit val templateRenderer = app.injector.instanceOf[TemplateRenderer]
```
Comment (jpatton638) [inline test/views/html/address/PostalAddressViewSpec.scala:35]: Could use "injected" method here

**Review by jpatton638 — APPROVED** [review]

File: test/views/html/address/MainAddressViewSpec.scala (hunk 1 above)
Comment (AndrewArkless) [reply test/views/html/address/MainAddressViewSpec.scala]: Changed

File: test/views/html/address/PostalAddressViewSpec.scala (hunk 2 above)
Comment (AndrewArkless) [reply test/views/html/address/PostalAddressViewSpec.scala]: CHanged

**Pull request merged**